tests/testthat/test-cut.R

```diff
@@ -0,0 +1,397 @@
+context("cut function")
+
+with_mock_crunch({
+    ds <- loadDataset("test ds")
+    gen <- ds$gender
+
+    test_that("cut labels are generated correcty", {
+        expect_identical(
+            crunch:::generateCutLabels(5, c(2.111111, 3, 4, 5), 4, FALSE, FALSE),
+            c("[2.1111,3)", "[3,4)", "[4,5)")
+        )
+        expect_identical(
+            generateCutLabels(2, c(2.111111, 3, 4, 5), 4, FALSE, FALSE),
+            c("[2.1,3)", "[3,4)", "[4,5)")
+        )
+    })
+
+    test_that("cut throws error when no variable name supplies", {
+        expect_error( cut(ds$birthyr, 3), "Must provide the name for the new variable")
+    })
+
+    test_that("cut labels are generated correctly", {
+        expect_identical( c("[2,3)", "[3,4)", "[4,5)"),
+            generateCutLabels(2, c(2, 3, 4, 5), 4, FALSE, FALSE)
+        )
```

Comment:
This is purely stylistic with `expect_identical` because of the symmetrically of identity, but usually the code being tested is the first argument and the check of it is the second.